src/zenflows/gql/mw/sign.ex

```diff
@@ -25,6 +25,8 @@ Absinthe middleware to verify GraphQL calls.
 alias Zenflows.Restroom
 alias Zenflows.VF.Person
 
+@missing_headers_auth_call "gql_user, gql_sign and gql_body headers are required for an authenticated call"
```

Comment:
They are not headers.  They are context variables.  They are set in a Plug.  Please either mention that "zenflows-user or zenflows-sign is missing" or leave it as it is.

Reply:
Yes you are right, but it is missing because we are trying to do an authenticated call, so would it be better "zenflows-user and zenflows-sign are required for an authenticated call"?

Reply:
"couldn't aunthenticate: zenflows-user and/or zenflows-sign headers are missing" sounds better.